### ISSUE TITLE
Multi-station rides only play music at one station.

### ISSUE THREAD

**Issue by AnAfterImage** [OpenRCT2/OpenRCT2#9605]
For example if you place a chairlift and set it to play music, it'll only play music at the first station, leaving the other station mute.

**Comment by Generalcamo** [comment]
This is an original bug.

**Comment by ocalhoun6** [comment]
And is it even a bug, per se? Seems like it was probably meant to be this way.

Still, though, playing music at all stations when selected might be a feature to request. Where I could see that being useful is if you want the same music throughout pretty much all the park, but you don't want to hear the cacophony where two music-playing rides overlap out of sync with one another. If you had four stations all playing the same music in sync with each other, though, it could sound good all over a medium-sized park.

**Comment by AnAfterImage** [comment]
Yeah I suppose it should be labelled as a feature request. It's kind of hard to have syncing music throughout the park(what with them starting up as soon as you select the music), unless you restart I suppose.

**Comment by ocalhoun6** [comment]
Try turning on the music while the game is paused, then un-pausing the game. That might get it to sync up ... or it might not.

At any rate, though, the music will de-sync whenever the rides break down, so you'd want to do this with a ride type that never breaks down.

**Comment by IntelOrca** [comment]
How about only playing music at stations with an entrance?

**Comment by AnAfterImage** [comment]
I suppose that could work. Would at least give you some control over what stations play music and which don't.
